jrnl/install.py

```diff
@@ -75,6 +75,12 @@ def load_or_install_jrnl(alt_config_path):
         logging.debug("Reading configuration from file %s", config_path)
         config = load_config(config_path)
 
+        if config is None:
+            print(
+                f"Configuration file {config_path} is empty, now using default config"
+            )
+            config = get_default_config()
```

Comment:
Rather than reverting the default config, I think jrnl should just exit. We don't know what exactly the user wants and I'd prefer not to make an assumption that they'd want to use their default config at this point.

Reply:
You're right about that. It's better to let the user decide what to do and resolve the problem first.